Examples/RegistrationITKv4/ImageRegistration6.cxx

```diff
@@ -411,26 +410,27 @@ int main( int argc, char *argv[] )
   //  The second image is the result of intentionally rotating the first
   //  image by $10$ degrees and shifting it $13mm$ in $X$ and $17mm$ in
   //  $Y$. Both images have unit-spacing and are shown in Figure
-  //  \ref{fig:FixedMovingImageRegistration5}. The registration takes $22$
+  //  \ref{fig:FixedMovingImageRegistration5}. The registration takes $21$
   //  iterations and produces:
   //
   //  \begin{center}
   //  \begin{verbatim}
-  //  [0.17429, 111.172, 131.563, 12.4582, 16.0724]
+  //  [ 0.174527, 12.4528, 16.0766]
   //  \end{verbatim}
   //  \end{center}
   //
   //  These parameters are interpreted as
   //
   //  \begin{itemize}
-  //  \item Angle         =                  $0.17429$     radians
-  //  \item Center        = $( 111.172    , 131.563      )$ millimeters
-  //  \item Translation   = $(  12.4582   ,  16.0724     )$ millimeters
+  //  \item Angle         =                  $0.174527$     radians
+  //  \item Translation   = $( 12.4528, 16.0766 )$ millimeters
   //  \end{itemize}
   //
+  // THIS EXPLANATION NEEDS TO BE CORRECTED SINCE WE ARE GETTING THE
+  //  SAME RESULTS WITHOUT THE CENTERED TRANSFORM
   //  Note that the reported translation is not the translation of $(13,17)$
```

Comment:
This section needs to be rewritten to discuss the cause of the translation being different has to to do with the fixed parameters.